build.gradle

```diff
@@ -1,29 +1,22 @@
 plugins {
-    id 'fabric-loom' version '0.2.6-SNAPSHOT'
+    id 'fabric-loom' version '0.8-SNAPSHOT'
     id "maven-publish"
     id "signing"
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_8
-targetCompatibility = JavaVersion.VERSION_1_8
+sourceCompatibility = JavaVersion.VERSION_16
+targetCompatibility = JavaVersion.VERSION_16
 
 archivesBaseName = project.archives_base_name
 version = project.mod_version
 group = project.maven_group
 
-minecraft {
-    refmapName = "refmap.reach-entity-attributes.json"
-}
-
 dependencies {
     minecraft "com.mojang:minecraft:${project.minecraft_version}"
     mappings "net.fabricmc:yarn:${project.yarn_mappings}"
     modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
 
-    modImplementation("net.fabricmc.fabric-api:fabric-resource-loader-v0:${project.resource_loader_version}") {
-        transitive false
-    }
-
+    modImplementation fabricApi.module("fabric-resource-loader-v0", project.fabric_version)
```

Comment:
The resource loader dependency should still use an explicit dependency to reduce volatility and increase clarity.

Reply:
I am not sure what you mean here ? I don't see how I could make the dependency more explicit.

Reply:
I was meaning to keep it as-is, with an explicit version manually retrieved from a POM, but I guess it doesn't matter either way. I just find the `fabricApi.module` to be rather hacky and counterintuitive in design.